### PR TITLE
docs: how to contribute to template

### PR DESCRIPTION
**Pull request by eoaksnes**
## Why is this pull request needed?

This pull request is needed because we want to provide potential project contributors with a short guide to how they can help with the project.

## What does this pull request change?

* Add CONTRIBUTING.md file 
* Links from README.md to CONTRIBUTING.md file
* Renamed how to commit inside docs to feature flags (since that information is moved to CONTRIBUTING.md file)

## Issues related to this change:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+2. Clone the project to your machine
+3. To keep track of the original repository add another remote named upstream
+```shell
+git remote add upstream git@github.com:equinor/template-fastapi-react.git
+```
+4. Create a branch locally with a succinct but descriptive name. 
+```shell
+git checkout -b improve-feature
+```
+4. Make the changes in the created branch.
```
Comment (sebastianvitterso) [inline CONTRIBUTING.md:56]: There are two `4.`-points.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+8. Before you send the pull request, be sure to rebase onto the upstream source. This ensures your code is running on the latest available code.
+```shell
+git fetch upstream
+git rebase upstream/main
+```
+9. Push to your fork.
+```shell
+git push origin improve-feature
+``` 
+9. Submit a pull request to the original repository (via Github interface). Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us.
```
Comment (sebastianvitterso) [inline CONTRIBUTING.md:80]: Two `9.`s too!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+## Your First Contribution
+
+You can start by looking through these beginner and help-wanted issues.
+
+## How to report a bug
+
+Report something that is broken or not working as
+intended [here](https://github.com/equinor/template-fastapi-react/issues/new?assignees=&labels=type%3A+%3Abug+bug&template=bug-report.md&title=).
+
+If you find a security vulnerability, do NOT open an issue.
```
Comment (sebastianvitterso) [inline CONTRIBUTING.md:95]: Possibly say what to do instead? E.g. "contact us via email etc.".

**Review by sebastianvitterso — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (eoaksnes) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (eoaksnes) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (eoaksnes) [reply CONTRIBUTING.md]: Updated

**Review by sebastianvitterso — APPROVED** [review]

**Pull request merged**